understanding/21/text-spacing.html

```diff
@@ -17,22 +17,23 @@ <h2>In brief</h2>
   </section>
   <section id="intent">
     <h2>Intent</h2>
-    <p>The intent of this Success Criterion (SC) is to ensure that when people override author specified text spacing to improve their reading experience, content is still readable and operable. Each of the requirements stipulated in the SC's four bullets helps ensure text styling can be adapted by the user to suit their needs.</p>
-    <p>The specified metrics set a minimum baseline. The values in between the author's metrics and the metrics specified in this SC should not have loss of content or functionality.</p>
+    <p>The intent of this Success Criterion (SC) is to ensure that when people override author-specified text spacing to improve their reading experience, content is still readable and operable. Each of the requirements stipulated in the SC's four bullets helps ensure text styling can be adapted by the user to suit their needs.</p>
+    <p>The metrics set a target for a minimum increase in text spacing that must be met. Starting from the author's baseline implementation, any increase in spacing for any of these four style properties up to and including these targets should not result in the loss of content or functionality.</p>
```

Comment:
```suggestion
    <p>The metrics set a target for a minimum for text spacing that must be met. Starting from the author's baseline font size, any increase in any of these four style properties up to and including these targets should not result in the loss of content or functionality.</p>
```

Reply:
Suggest: Starting from the author's baseline font size, any change to any of these four style properties to the specified values should not result in a loss of content or functionality.